Program.cs

```diff
@@ -177,6 +178,10 @@ public ValidationResult DotnetResult
       set => this.SetProperty(ref _dotnetResult, value);
     }
 
+    private CancellationTokenSource validatorCancellationSource = null;

```

Comment:
```suggestion
    private CancellationTokenSource _validatorCancellationSource;
```

It seems the C# convention is to start private fields with an underscore.

Don't have to set it to `null`, C# runtime does that by default (different from C++)